packages/server/src/controllers/chat-messages/index.ts

```diff
@@ -158,6 +166,8 @@ const removeAllChatMessages = async (req: Request, res: Response, next: NextFunc
         const _chatType = req.query?.chatType as string | undefined
         const startDate = req.query?.startDate as string | undefined
         const endDate = req.query?.endDate as string | undefined
+        const startDateTime = req.query?.startDateTime as string | undefined
```

Comment:
why do we need `startDateTime` and `endDateTime` since we already have `startDate` and `endDate`? wouldnt that be confusing which to use?

Reply:
I see, the new ones include time. I think for consistency, we can just use `startDate`and `endDate` but with precision down to time, instead of just dates

Reply:
Yeah I like that, I didn't want to change the API much in case others are using it but that would be much better. 

We're looking for querying down to the minute level so whatever gets us there is perfect!